src/index.js

```diff
@@ -49,20 +30,13 @@ function articleDiv(article, articleTemplateSelector, options) {
 
   var articleImage;
 
-  if (
-    article._embedded &&
-    article._embedded["wp:featuredmedia"] &&
-    article._embedded["wp:featuredmedia"][0] &&
-    article._embedded["wp:featuredmedia"][0].media_details &&
-    article._embedded["wp:featuredmedia"][0].media_details.width &&
-    article._embedded["wp:featuredmedia"][0].media_details.height
-  ) {
+  if (article._embedded && article._embedded["wp:featuredmedia"] && article._embedded["wp:featuredmedia"][0] && article._embedded["wp:featuredmedia"][0].media_details && article._embedded["wp:featuredmedia"][0].media_details.width && article._embedded["wp:featuredmedia"][0].media_details.height) {
     var media = article._embedded["wp:featuredmedia"][0];
```

Comment:
or even better define it with optional chaining above line 33 and then use it everywhere below